### DISCUSSION TITLE
Need some files.

### DISCUSSION THREAD

**Discussion by liuyang-3520** [endink/Mediapipe4u-plugin#140] [General]
I can not download some files. Please send me them directly. Thank you very much. File name list: 
MediaPipe4U, MediaPipe4UBVH, MediaPipe4ULiveLink, MediaPipe4ULLM, MediaPipe4ULLMSpeech, MediaPipe4UNvAR

**Comment by endink** [comment]
Theses are folders, not files, please read github page to find out download link
